python/lsst/afw/table/Base.i

```diff
@@ -609,6 +610,7 @@ aliases = {
     int: "I",
     float: "D",
     str: "String",
+    futurestr: "String",
```

Comment:
No, because there is code above AFW that has not been migrated, and also string literals in Python 2 will not be compatible with future `str` so we we would have to enable `unicode_literals` in all the caller code. This approach lets me support both during transition to python 3. When we drop python2 this line can go away.